ldk/javascript/src/types/oliveHelps/index.d.ts

```diff
@@ -340,12 +341,14 @@ declare namespace OliveHelps {
 
   type DateTimeType = 'date' | 'time' | 'date_time';
 
+  type IconSize = 'small' | 'medium' | 'large' | 'x-large';
+
+  type Color = 'white' | 'grey' | 'black' | 'whisper-strip';
```

Comment:
Note that `black` and `whisper-strip` are not possibilities of the `SectionTitle` component. In those cases, if those colors a chosen by Loop Author it will currently fallback to its default color. This Color property is TBD

Reply:
Just to confirm - that is just an interim implementation until we figure out what changes we want to make to our approach to colour, right? Otherwise I really don't think we should support someone assigning invalid values to a component.

Reply:
So if you planing to add extra colors to the SectionTitle component, do you need to add mapping for them in the Sidekick? If not then all unmapped values will be defaulted to 'white'.

Reply:
`white` is currently the default value for `SectionTitle`. We will discuss this with UX.